runecoral/runecoral.cpp

```diff
@@ -100,13 +101,14 @@ RuneCoralLoadResult create_inference_context(const char *mimetype, const void *m
         tflite::InterpreterBuilder(*(context->model), context->resolver)(&(context->interpreter));
 
         if (context->interpreter) {
+            if (!accelerateInterpreter(backend, context)) {
+                LOG_E("Unable to accelerate interpreter");
```

Comment:
Do we want to error out here, or is it fine to just keep going without acceleration?

My understanding is that hardware acceleration is more of a hint than a command, so it *should* be okay.

Reply:
Yep, we will fall back to CPU based inference => which means all the inference will still happen.
The only time this can error out is when the backend is not available - which was already reported by `availableAccelerationBackends()`, so i think falling back is a safe option for now.

We still have to decide how the runes / saga can hint which acceleration backends they'd want to support though... but that is on `rune` 's side